File: swarmmanagercontroller/src/main/java/co/uk/swarmbit/docker/api/configs/parameters/ConfigsFilters.java
package co.uk.swarmbit.docker.api.configs.parameters;

import co.uk.swarmbit.docker.api.common.AbstractFilters;
import co.uk.swarmbit.docker.api.common.Filters;

public class ConfigsFilters extends AbstractFilters implements Filters {


    public ConfigsFilters setId(String id) {
        addFilter("id", id);
        return this;
    }

    public ConfigsFilters setName(String name) {
        addFilter("name", name);
        return this;
    }

    public ConfigsFilters setLabel(String label) {
        addFilter("label", label);
        return this;
    }

}

File: swarmmanagercontroller-new/src/main/kotlin/com/swarmbit/docker/api/configs/parameters/ConfigsFilters.kt
package com.swarmbit.docker.api.configs.parameters

import com.swarmbit.docker.api.common.AbstractFilters

class ConfigsFilters : AbstractFilters() {

    fun setId(id: String): ConfigsFilters {
        addFilter("id", id)
        return this
    }

    fun setName(name: String): ConfigsFilters {
        addFilter("name", name)
        return this
    }

    fun setLabel(label: String): ConfigsFilters {
        addFilter("label", label)
        return this
    }

}